### PR TITLE
Rename all imports to use correct module name and add vendoring

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

**Comment by fromanirh** [comment]
I suggest to fix the failing image lane (with a separate PR) before to move forward with this one

File: go.mod [hunk 1]
```diff
@@ -20,6 +20,200 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.2
 )
 
+require (
```
Comment (fromanirh) [inline go.mod:23]: I like this change, but let's also make sure to update the non-openshift Dockerfile to make use of the vendor directory (right now, per operator-sdk standard, it does not)

Comment (cynepco3hahue) [reply go.mod:23]: it is not my initiative, it is go version 1.17

**Comment by fromanirh** [comment]
/approve
/lgtm
thanks @cynepco3hahue !

**Pull request merged**